src/Controller/Controller.php

```diff
@@ -159,7 +163,7 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      *
      * @var bool
      */
-    public $autoRender = true;
+    protected $autoRender = true;
```

Comment:
This change is not BC and actually not meaningless. There could be some doing `$this->getController()->autoRender = false` in a component.